packages/mobile/src/consumerIncentives/ConsumerIncentivesHomeScreen.tsx

```diff
@@ -63,7 +64,7 @@ function useHasBalanceForSupercharge() {
     if (tokenUserInfo?.balance.gte(tokenConfig.minBalance)) {
       return {
         hasBalanceForSupercharge: true,
-        token: tokenUserInfo.symbol,
+        superchargingToken: tokenConfig,
```

Comment:
`superchargingToken` sounds like it will return the actual token, how about renaming to `superchargingTokenConfig`?

edit: I see that `useDefaultTokenToSupercharge` was also calling `token` to the `tokenConfig`. I do think that adding the `config` suffix makes it clearer, but I leave it up to you